packages/eslint-plugin/src/rules/prefer-find.ts

```diff
@@ -125,22 +125,22 @@ export default createRule({
       return isAtLeastOneArrayishComponent;
     }
 
-    function getObjectIfArrayAtExpression(
+    function getObjectIfArrayAtZeroExpression(
       node: TSESTree.CallExpression,
     ): TSESTree.Expression | undefined {
       // .at() should take exactly one argument.
       if (node.arguments.length !== 1) {
         return undefined;
       }
 
-      const atArgument = getStaticValue(node.arguments[0], globalScope);
-      if (atArgument != null && isTreatedAsZeroByArrayAt(atArgument.value)) {
-        const callee = node.callee;
-        if (
-          callee.type === AST_NODE_TYPES.MemberExpression &&
-          !callee.optional &&
-          isStaticMemberAccessOfValue(callee, 'at', globalScope)
-        ) {
+      const callee = node.callee;
```

Comment:
This change just reverses the order of whether .at, or the (0), are inspected first. It's not needed for the fix, but I found it surprising that `foo(Symbol.for('throws'))` would throw, since it's not even an `at` call; it's just any old function. WIth this change, you have to actually have 
`x.at(Symbol.for('throws'))` in order to trigger the faulty code.